module/User/src/Model/Enums/JWTClaims.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace User\Model\Enums;
+
+use Decision\Model\Member as MemberModel;
+use Laminas\Mvc\I18n\Translator;
+
+/**
```

Comment:
Fix this PHP_CS Warning. The enum related ones can be ignored as long as there is no support.

Reply:
I believe this is still an enum issue, it does not understand that the docblock belong to `enum` and not the file. Maybe we should just use the dev version of PHP_CS?